### PR TITLE
Code Climate Integration

### PR DESCRIPTION
**Pull request by vonvick**
#### What does this PR do?
Integrate code climate into the application
#### Description of Task to be completed?
Integrate code climate with the application
#### How should this be manually tested?
This can be manually tested by the code climate badge that is found un the readme file
#### Any background context you want to provide?
 No background context
#### What are the relevant pivotal tracker stories?
Add code climate to the application
#### Screenshots (if appropriate)
Not applicable
#### Questions:
No question

### CODE REVIEW

File: build/js/inverted-index.js [hunk 1]
```diff
@@ -14,154 +14,168 @@ class InvertedIndex {
...
-
   /**
-   * loops through an array and removes duplicate elements
-   * @static
-   * @param {}
-   * @returns {Array} 
+   * check the documents in the uploaded file and returns
+   * the content of the file in a tokenized format
+   * @param {Array} file the content of the file
+   * @returns {Array} returns a boolean
```
Comment (fortunee) [inline build/js/inverted-index.js:31]: Your returns specifies a type of Array while your descriptions says you're returning a Boolean. 
Rectify accordingly

File: build/js/inverted-index.js [hunk 2]
```diff
@@ -14,154 +14,168 @@ class InvertedIndex {
...
    */
-  readBookData(book) {
-    if(book.length < 1) {
+  checkBookData(file) {
+    if (!Array.isArray(file) || file.length < 1) {
       return false;
-    } else {
-      return true;
+    }
+    for (let i = 0; i < file.length; i += 1) {
```
Comment (fortunee) [inline build/js/inverted-index.js:55]: You could increment your iterator with this `i++` instead of `i += 1`

File: build/js/inverted-index.js [hunk 3]
```diff
@@ -14,154 +14,168 @@ class InvertedIndex {
...
+      if (!file[i].title || !file[i].text) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  /**
+   * loops through an array and removes duplicate elements
+   * @param {}
```
Comment (fortunee) [inline build/js/inverted-index.js:65]: Specify your param with descriptions accordingly

File: build/js/inverted-index.js [hunk 4]
```diff
@@ -14,154 +14,168 @@ class InvertedIndex {
...
+   * @returns {Array}
+   */
+  sortDocument(bookName, items, element) {
+    element = parseInt(element, 10);
+    for (let item = 0; item < items.length; item += 1) {
```
Comment (fortunee) [inline build/js/inverted-index.js:70]: Consider `i++` instead of `i+=1`

File: build/js/inverted-index.js (hunk 2 above)
Comment (vonvick) [reply build/js/inverted-index.js:55]: the ESLint rule for the checkpoint flags an error if the incrementor is used that way

File: build/js/inverted-index.js (hunk 4 above)
Comment (vonvick) [reply build/js/inverted-index.js:70]: the ESLint rule for the checkpoint flags an error if the incrementor is used that way

**Pull request merged**